src/Build.UnitTests/FileLogger_Tests.cs

```diff
@@ -368,6 +368,49 @@ public void EmptyErrorLogUsingWarningsErrorsOnly(string loggerOption)
             }
         }
 
+        /// <summary>
+        /// File logger is writting the verbosity level as soon the build starts.
+        /// </summary>
+        [Theory]
+        [InlineData(LoggerVerbosity.Quiet, false)]
+        [InlineData(LoggerVerbosity.Minimal, false)]
+        [InlineData(LoggerVerbosity.Normal, true)]
+        [InlineData(LoggerVerbosity.Detailed, true)]
+        [InlineData(LoggerVerbosity.Diagnostic, true)]
+        public void LogVerbosityMessage(LoggerVerbosity loggerVerbosity, bool shouldContain)
+        {
+            using (var testEnvironment = TestEnvironment.Create())
+            {
+                var fileLogger = new FileLogger
+                {
+                    Verbosity = loggerVerbosity
+                };
+
+                var logFile = testEnvironment.CreateFile(".log");
+                fileLogger.Parameters = "logfile=" + logFile.Path;
+
+                Project project = ObjectModelHelpers.CreateInMemoryProject(@"
+                <Project ToolsVersion=`msbuilddefaulttoolsversion` xmlns=`msbuildnamespace`>
+                    <Target Name=`Build` />
+                </Project>
+                ");
+
+                project.Build(fileLogger);
+                project.ProjectCollection.UnregisterAllLoggers();
+
+                string log = File.ReadAllText(logFile.Path);
+                var message = $"Logging verbosity is set to: {loggerVerbosity}.";
```

Comment:
This will fail on non-English OSes. It might not be the only test to do so but we try to support building and testing MSBuild on any language. We generally use

again in the tests.
```suggestion
                var message = ResourceUtilities.FormatResourceStringStripCodeAndKeyword("LogLoggerVerbosity", loggerVerbosity);
```

Reply:
Good catch again, thanks for the help - that is again super obviously 🙄 I had to know that better...